docs/operations/single-server.md

```diff
@@ -42,37 +43,43 @@ The startup scripts for these example configurations run a single ZK instance al
 
 The example configurations run the Druid Coordinator and Overlord together in a single process using the optional configuration `druid.coordinator.asOverlord.enabled=true`, described in the [Coordinator configuration documentation](../configuration/index.md#coordinator-operation).
 
+The `start-druid` is a generic launch script for starting druid services on single server, it accepts optional arguments like services, memory and config.
+All reference configurations can be acheived by passing appropriate arguments to this script.
+Existing launch scripts are deprecated and will be removed in the next release. 
```

Comment:
This should have a better word than "Existing", maybe "Other"? After all when someone reads this the `bin/start-druid` will exist also

Reply:
+1

Reply:
> Existing launch scripts are deprecated and will be removed in the next release.

I'd rather not include this line ☝️ at all, because IMO we should make the decision to drop the old scripts separately from adding this new feature. Also, never promise anything will happen in "the next release". It's almost a guarantee that it won't happen 🙂